### PR TITLE
Add active/inactive feature on initiatives

### PR DESCRIPTION
**Pull request by iamlasse**
Ok last update today, inactive/active state on Initiative

### CODE REVIEW

File: client/templates/initiative/card/initiativeCard.js [hunk 1]
```diff
@@ -12,5 +22,13 @@ Template.initiativeCard.events({
     e.preventDefault();
     var initiative = this;
     addOrRemoveVote(initiative); 
+  },
+  'click .status': function(e, tpl){
+    Meteor.call('setInactiveActive', this, function(err, res){
+      if(err)
+        sAlert.error(err.message);
+
```
Comment (trustieee) [inline client/templates/initiative/card/initiativeCard.js]: probably want an else here, so we don't show an error and say it was updated at the same time.

Comment (iamlasse) [inline client/templates/initiative/card/initiativeCard.js]: Yes good point lol

File: client/templates/initiatives/initiatives.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ Template.initiatives.helpers({
       }), cursor);
```
Comment (trustieee) [inline client/templates/initiatives/initiatives.js:7]: When using the search box, there is no logic applied for only retrieving active Initiatives. We should apply that same logic to it.

Comment (iamlasse) [inline client/templates/initiatives/initiatives.js:7]: Ahh good point.. It feels not very dry.. I am typing same code in several places... wonder how we can dry it out... Do something like Initiatives.active().find() would collection helpers be able to help?

**Comment by trustieee** [comment]
I think a collection helper may do the trick. Or only publish active Initiatives for this set displayed on the landing page.

**Pull request merged**